fare/config.py

```diff
@@ -151,3 +151,13 @@ def _(x):
 
 #: Switches off incept of redirects by Flask-DebugToolbar.
 DEBUG_TB_INTERCEPT_REDIRECTS = False
+
+FIXTURES_FILES_LOCATION = 'data/'
```

Comment:
This has to be written in the README file. It's very important since when we will have the prod deployment we will need to configure a volume for this to maintain persistence